blueocean-commons/src/main/java/io/jenkins/blueocean/commons/analytics/Analytics.java

```diff
@@ -71,7 +71,7 @@ public void track(TrackRequest req) {
     protected abstract void doTrack(String name, Map<String, Object> allProps);
 
     protected final String server() {
-        return InstanceIdentity.get().getPublic().getPublicExponent().toString(16);
+        return Hashing.md5().hashBytes(InstanceIdentity.get().getPublic().getEncoded()).toString();
```

Comment:
hmm, I thought RSA public key generated using sha256 algorithm in Jenkins, not sure though. regardless I think safer to use sha-256.